pegasus/sites.v3/code.org/views/dance_teacher_resources.haml

```diff
@@ -0,0 +1,32 @@
+:ruby
+  resources = [
+    {
+      title: "Dance Party",
+      description: "Code a Dance Party to share with your friends!",
+      image: "/images/animated-examples/cutesprite.gif",
+      button_text: "Start",
+      link: CDO.studio_url('/s/dance')
+    },
+    {
+      title: "Keep Dancing",
+      description: "Go beyond the first hour with extended Dance Party project ideas.",
+      image: "/images/animated-examples/cutesprite.gif",
+      button_text: "Go Beyond",
+      link: CDO.studio_url('/s/dance-extras')
+    },
+  ]
+
+.resources
+  -resources.each do |resource|
+    .col-50
+      .resource-card{style: "height: 234px; width: 468px; border: 1px solid gray; margin: 20px; overflow: hidden; position: relative; float: left;"}
```

Comment:
we will probably want to pull all of the styles in this partial into a stylesheet

Reply:
Good call. Done!